middleman_protocol/middleman_protocol/concent_golem_messages/message.py

```diff
@@ -21,9 +23,16 @@ class NonceAbstractMessage(Message):
         'nonce',
     ] + Message.__slots__
 
+    def slots(self) -> List[List[Union[str, bytes, int]]]:
+        slots = super(NonceAbstractMessage, self).slots()
+        if ['nonce', self.nonce] in slots:  # pylint: disable=no-member
+            slots[slots.index(['nonce', self.nonce])][1] = str(self.nonce)  # pylint: disable=no-member
+        return slots
+
     def deserialize_slot(self, key: str, value: Any) -> Any:
         value = super().deserialize_slot(key, value)
         if key == 'nonce':
+            value = int(value)
```

Comment:
If you apply comment above you can remove this.

Reply:
Added comment in code